.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
I don't have enough bandwidth to address everyones concerns about editorconfig, and frankly, don't care. I'm going to back out this file-- leaving only VS code settings. Use https://github.com/aspnet/Universe/issues/500 for more discussion on this.